pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -9,12 +9,19 @@ const (
 	SecurityContextConstraintPrivileged = "privileged"
 	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
 	SecurityContextConstraintRestricted = "restricted"
+	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
+	SecurityContextConstraintNonRoot = "nonroot"
+	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
+	SecurityContextConstraintHostMount = "hostmount"
```

Comment:
We actually should have descriptions on all of these that describe exactly
what they do, and how insecure they are.  Like giant warning kind of output.

On Thu, Oct 29, 2015 at 1:45 PM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> Hrm - we could really use a description on this one...
>
> On Thu, Oct 29, 2015 at 1:42 PM, Jordan Liggitt <notifications@github.com>
> wrote:
>
>> In pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go
>> <https://github.com/openshift/origin/pull/5498#discussion_r43418964>:
>>
>> > @@ -9,12 +9,19 @@ const (
>> >  	SecurityContextConstraintPrivileged = "privileged"
>> >  	// SecurityContextConstraintRestricted is used as the name for the system default restricted scc.
>> >  	SecurityContextConstraintRestricted = "restricted"
>> > +	// SecurityContextConstraintNonRoot is used as the name for the system default non-root scc.
>> > +	SecurityContextConstraintNonRoot = "nonroot"
>> > +	// SecurityContextConstraintHostMount is used as the name for the system default host mount only scc.
>> > +	SecurityContextConstraintHostMount = "hostmount"
>>
>> PV recycler pods need it and nothing else (afaik)
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openshift/origin/pull/5498/files#r43418964>.
>>
>
>